tests/test_lpass.py

```diff
@@ -4,61 +4,47 @@
 
 
 class TestLPass(unittest.TestCase):
-    @patch('db_facts.lpass.check_output')
-    def test_lpass_field_notes(self, mock_check_output):
-        mock_check_output.return_value = "fakenotes\n".encode("utf-8")
-        out = lpass.lpass_field('my_name', 'notes')
-        mock_check_output.\
-            assert_called_with(['lpass', 'show', '--notes', 'my_name'])
-        assert out == "fakenotes"
-
     @patch('db_facts.lpass.check_output')
     def test_lpass_field_username(self, mock_check_output):
         mock_check_output.return_value = "fakeuser\n".encode("utf-8")
         out = lpass.lpass_field('my_name', 'username')
-        mock_check_output.\
-            assert_called_with(['lpass', 'show', '--username', 'my_name'])
+        mock_check_output.assert_called_with(
+            ['op', 'item', 'get', 'my_name', '--field', 'label=username'])
         assert out == "fakeuser"
 
     @patch('db_facts.lpass.check_output')
     def test_lpass_field_password(self, mock_check_output):
         mock_check_output.return_value = "fakepassword\n".encode("utf-8")
         out = lpass.lpass_field('my_name', 'password')
-        mock_check_output.\
-            assert_called_with(['lpass', 'show', '--password', 'my_name'])
+        mock_check_output.assert_called_with(
+            ['op', 'item', 'get', 'my_name', '--field', 'label=password'])
         assert out == "fakepassword"
 
-    @patch('db_facts.lpass.check_output')
-    def test_lpass_field_url(self, mock_check_output):
-        mock_check_output.return_value = "fakeurl\n".encode("utf-8")
-        out = lpass.lpass_field('my_name', 'url')
-        mock_check_output.\
-            assert_called_with(['lpass', 'show', '--url', 'my_name'])
-        assert out == "fakeurl"
```

Comment:
same deal as with the Notes field test function -- I'd prefer for this to throw an error rather than be deleted entirely